pwndbg/aglib/stack.py

```diff
@@ -176,16 +178,22 @@ def _fetch_via_exploration() -> Dict[int, pwndbg.lib.memory.Page]:
     return stacks
 
 
-def callstack() -> List[int]:
+def callstack() -> Generator[int, None, None]:
```

Comment:
Also I am not super convinced, but maybe we want to keep callstack returning a list and have a callstack_iter function.